docs/sql/cli.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: SQL CLI
+parent: SQL
+nav_order: 20
+---
+
+# SQL CLI
+
+SQL CLI is a stand-alone Python application that you can launch with the `odfesql` command.
+
+Install the SQL plugin to your Elasticsearch instance, run the CLI using MacOS or Linux, and connect to any valid Elasticsearch end-point.
+
+![SQL CLI](../../images/cli.gif)
+
+## Features
+
+SQL CLI has the following features:
+
+- Multi-line input
+- Autocomplete for SQL syntax and index names
+- Syntax highlighting
+- Formatted output:
+  - Tabular format
+  - Field names with color
+  - Enabled horizontal display (by default) and vertical display when output is too wide for your terminal, for better visualization
+  - Pagination for large output
+- Works with or without security enabled
+- Supports loading configuration files
+- Supports all SQL plugin queries
+
+## Install
+
+To install the CLI:
+
+1. We suggest you install and activate a python3 virtual environment to avoid changing your local environment:
+
+```
+pip install virtualenv
+virtualenv venv
+source ./bin/activate
+```
+
+2. To install the CLI:
+
+```
+pip install odfesql
+```
+
+The SQL CLI only works with Python 3.
+{: .note }
+
+## Using the CLI
+
+1. Index the sample [accounts test data](https://github.com/opendistro-for-elasticsearch/sql/blob/master/src/test/resources/doctest/testdata/accounts.json).
+1. To launch the CLI, run:
+```
+odfesql
+```
+1. Run a sample SQL command:
+```
+SELECT * FROM accounts
+```
+By default, you see a maximum output of 200 rows. To show results more than 200 rows, add a `LIMIT` clause with the desired value.
+
+## Configure
+
+When you first launch the SQL CLI, a configuration file is automatically created at `~/.config/odfesql-cli/config` (for MacOS and Linux). After you configure the SQL CLI the first time, it's auto-loaded thereafter.
+
+You can configure the following connection properties:
+
+- `endpoint`: You do not need to specify an option, anything that follows the launch command `odfesql` is considered as the endpoint. If you do not provide an endpoint, by default, the SQL CLI connects to http://localhost:9200.
+- `-u/-w`: Supports username and password for HTTP basic authentication, such as with the security plugin or fine-grained access control for Amazon Elasticsearch Service.
+
```

Comment:
Missing an auth option here `--aws-auth`. 
"--aws-auth: turn on to use AWS sigV4 authentication. It can be configured by AWS CLI `aws configure` command. If provided with this option, ODFE SQL CLI will try to retrieve local aws configuration to authenticate and connect" 
Basically, this option is only needed when user connects to Amazon Elasticsearch endpoint

Reply:
Added.

Reply:
It should be `--aws-auth`.